translations/en.json

```diff
@@ -1106,7 +1106,7 @@
     "wallet-manage-assets": "Manage assets",
     "wallet-offline": "Wallet offline",
     "wallet-request": "Request",
-    "wallet-send": "Send",
+    "wallet-send": "Next",
```

Comment:
we can't just change it in translation because we have same UI for sending in chat, but we send a message there